libc/src/__support/FPUtil/generic/FMod.h

```diff
@@ -174,13 +174,11 @@ class FMod {
     const T quiet_nan = FPB::quiet_nan().get_val();
     FPB sx(x), sy(y);
     if (LIBC_LIKELY(!sy.is_zero() && !sy.is_inf_or_nan() &&
-                    !sx.is_inf_or_nan())) {
+                    !sx.is_inf_or_nan()))
       return false;
-    }
 
     if (sx.is_nan() || sy.is_nan()) {
-      if ((sx.is_nan() && !sx.is_quiet_nan()) ||
-          (sy.is_nan() && !sy.is_quiet_nan()))
+      if (sx.is_signalling_nan() || sy.is_signalling())
```

Comment:
Should this be using either `is_signalling_nan` or `is_signalling` for BOTH `sx` AND `sy`?

Reply:
Actually neither of them are correct.  The correct function is `is_signaling_nan()`.